manifests/init.pp

```diff
@@ -254,6 +258,15 @@
     }
   }
 
+  if $pulp3 and $pulp_master {
+    class { 'pulpcore':
+      manage_apache => false,
+    }
+    foreman::config::apache::fragment { 'pulp':
+      ssl_content => file('foreman_proxy_content/pulpcore-apache.conf.erb'),
```

Comment:
I think we want to use `template` instead of `file` here.

Reply:
done, thanks